### ISSUE TITLE
update #2

### ISSUE THREAD

**Issue by loreese** [loreese/sherlock#4]
We ran into some issues this week with nailing down a quantifiable question and decided to change direction with our analysis of Sherlock. Instead of looking at representations of disability in a few short stories, we are now considering *The Sign of the Four* through an Orientalism lens as we decided there simply wasn't a significant enough employment of disability in the corpus to move forward. With the text chosen, we will be marking up a chapter of the text by tagging oriental objects and oriental elements assigned to characters. Our working question now is something to the effect of: How is the body "orientalized" in this text? To illuminate this, we think we will be analyzing speech and how the Orient is talked about and how people are talked to.

**Comment by sjw82** [comment]
This is an exciting new lens to use!

If I could offer a resource, there is a book that they always highly suggested we read if our senior thesis was on anything Eastern when we were in high school. It is about the West's treatment and ideology of Eastern cultures. I don't know that it'll be pertinent to your research but its good for researchers to have some background on it, even if it's just from the sparknotes.

The wiki: https://en.wikipedia.org/wiki/Orientalism_(book)
The book: https://monoskop.org/images/4/4e/Said_Edward_Orientalism_1979.pdf

**Comment by loreese** [comment]
@sjw82 agreed, I definitely had Said in mind with this question of the oriental body. Thanks for the links!

**Comment by Ethandaniel47** [comment]
I assume you'll be referencing Edward Said's "Orientalism"for your research? In an attempt to tie in certain aspects of the original research questions, have you considered marking stereotypical "oriental" traits in certain characters in the text, both physical, emotional, and concerning dress?

**Comment by MJB288** [comment]
I find this to be an interesting approach to take. Out of curiosity, what led you to choosing this particular question on short notice over others? Also, depending upon how your markup of <i>The Sign of the Four</i> goes, would you be extending this to other Sherlock Novels as well (I don't know enough about Sherlock Novels to know if this would even be feasible)?

Ultimately, I'm glad you realized this now. There is nothing more disheartening than going through a project for a good chunk of the semester only to find out that it won't meet the expectations of the class. I wish you good luck on your new task and hope it is suitable enough for the project grade.

**Comment by rmf96** [comment]
I really like the new direction that you are taking your project! I know that it's difficult some times to put your research goals in to practice, as our group had some very similar problems conceptualizing exactly what it was that we wanted to look at and how we were going to accomplish that. I'm very excited to see where this goes.

**Comment by BenBavar** [comment]
The Sign of the Four (apparently cannot italicize on my phone) was published over a century ago. That makes me wonder to what extent what we commonly regard as "oriental" today lines up with what was (regarded as) characteristic of the Orient back then. Will you do some historical research to make sure you are not missing elements that Doyle could have seen as orientalizing?
